vite.config.ts

```diff
@@ -19,7 +19,14 @@ export default defineConfig({
   server: {
     port: 3000,
     open: true,
-    host: true
+    host: true,
+    proxy: {
+      '/api': {
+        target: 'http://localhost:8080',
```

Comment:
we have a variable in env for api path, can you use it here?

Reply:
> we have a variable in env for api path, can you use it here?

`.env` will be changed after this pr so I can just create an extra variable for `.env` but not sure if that makes sense